CHANGELOG.md

```diff
@@ -58,6 +58,8 @@
 
 ### 🚀 New components 🚀
 
+- `schemaprocessor`: Starting the initial work to allow from translating from semantic convention to another (#8371)
```

Comment:
I would rebase this if I could, but we had another release and this needs to be moved up again. Sorry about this.